injectionapi/core/injectionrequestfactory.go

```diff
@@ -1,8 +1,8 @@
 package core
 
 import (
-	"github.com/socketlabs/socketlabs-go/injectionapi/core/serialization"
-	"github.com/socketlabs/socketlabs-go/injectionapi/message"
+	"github.com/PraneethChandraThota/socketlabs-go/injectionapi/core/serialization"
+	"github.com/PraneethChandraThota/socketlabs-go/injectionapi/message"
```

Comment:
after your tests are complete please return the imports back to the main repo

Reply:
Changed back to the main repo